src/api.js

```diff
@@ -1,17 +1,28 @@
 import axios from "axios";
+
+const Ncnews = axios.create({
```

Comment:
Great that you're making use of an axios instance to set up a `baseUrl`, so you only have to specify it in one place in your code 🙌 
I would stick to camelCase when naming it though, as capitalised variables usually indicate it being a class